jujugui/static/gui/src/app/components/shared/status/status/status.js

```diff
@@ -94,8 +98,8 @@ class Status extends React.Component {
     @returns {Object} The resulting element.
   */
   _generateRemoteApplications() {
-    const {remoteApplications} = this.props.entities;
-    if (remoteApplications.length === 0) {
+    const {remoteApplications} = this.props.valueStore;
+    if (!Object.keys(remoteApplications).length) {
```

Comment:
is `remoteApplications` always defined in the `valueStore`? Same Q for the remaining sections of this file.

Reply:
Yep, Maraca always returns the top level keys.